frontend/src/component/project/Project/ProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -274,13 +292,31 @@ export const ProjectFeatureToggles = ({
                     value: boolean;
                     row: { original: ListItemType };
                 }) => (
-                    <FeatureToggleSwitch
-                        value={value}
-                        projectId={projectId}
-                        featureName={feature?.name}
-                        environmentName={name}
-                        onToggle={onToggle}
-                    />
+                    <>
+                        <FeatureToggleSwitch
+                            value={value}
+                            projectId={projectId}
+                            featureName={feature?.name}
+                            environmentName={name}
+                            onToggle={onToggle}
+                        />
+                        <ConditionallyRender
+                            condition={
+                                hasOtherEnvsWithVariants(feature?.name) &&
+                                feature.environments[name]?.variantCount ===
+                                    0 &&
+                                feature.environments[name].enabled
```

Comment:
Could maybe be simplified and extracted into a function that takes the feature and environment (name), but I'm not sure if it would add much value.

Reply:
Yes, I wanted to extract it to a constant and keep it close, but it was not possible in this context. I'm happy to jump into another pairing session to make this better